platform/linuxbsd/wayland/display_server_wayland.cpp

```diff
@@ -1042,6 +1081,138 @@ void DisplayServerWayland::window_start_resize(WindowResizeEdge p_edge, WindowID
 	wayland_thread.window_start_resize(p_edge, p_window);
 }
 
+void DisplayServerWayland::window_set_hdr_output_enabled(const bool p_enabled, WindowID p_window) {
```

Comment:
You probably want to remember if the user requested HDR be enabled so you can turn it on for them when HDR becomes available.